### ISSUE TITLE
Create a Dashboard with Project Status and flow

### ISSUE THREAD

**Issue by hmanalai** [BCStudentSoftwareDevTeam/urcpp#36]
**[Original report](https://bitbucket.org/laborstudents/urcpp-flask/issue/36) by Scott Heggen (Bitbucket: [sheggen](https://bitbucket.org/sheggen), GitHub: [sheggen](https://github.com/sheggen)).**

----------------------------------------

On the main page, make it clearer what state they are in:

* Incomplete
* Complete (Pending)
* Approved/Denied
* Withdrawn

Also give them the ability to withdraw their own application.

Looking further down the road: after an application is withdrawn, should they be able to re-apply? (I'm willing to bet that's what the user will think it means)

**Comment by hmanalai** [comment]
**Original comment by Guillermo ramos (Bitbucket: [guillermo_ramos](https://bitbucket.org/guillermo_ramos), GitHub: [memo330179](https://github.com/memo330179)).**

----------------------------------------

Should this be on the front page or the review for the application?

**Comment by hmanalai** [comment]
**Original comment by Scott Heggen (Bitbucket: [sheggen](https://bitbucket.org/sheggen), GitHub: [sheggen](https://github.com/sheggen)).**

----------------------------------------

I think it makes sense on the review page.

**Comment by hmanalai** [comment]
**Original comment by Guillermo ramos (Bitbucket: [guillermo_ramos](https://bitbucket.org/guillermo_ramos), GitHub: [memo330179](https://github.com/memo330179)).**

----------------------------------------

add status on review page see #36

**Comment by hmanalai** [comment]
**Original comment by Guillermo ramos (Bitbucket: [guillermo_ramos](https://bitbucket.org/guillermo_ramos), GitHub: [memo330179](https://github.com/memo330179)).**

----------------------------------------

![Screenshot from 2016-10-03 14-11-07.png](https://bitbucket.org/repo/e47RB5/images/3998758754-Screenshot%20from%202016-10-03%2014-11-07.png)

Does this take care of it?

**Comment by hmanalai** [comment]
**Original comment by Scott Heggen (Bitbucket: [sheggen](https://bitbucket.org/sheggen), GitHub: [sheggen](https://github.com/sheggen)).**

----------------------------------------

1) Is it prominent enough (am I going to get questions like "What's the status of my application?" because they don't see this)? Maybe it should also be on the front page (I am fully aware of my self-contradiction by stating this); maybe the front page needs reworked entirely? Something for us to discuss.

2) How do they get back to the review page from the front page? Is the link changed after submission is completed? 

3) Will the user automatically know what "Pending" means? Would "Pending; awaiting review by committee" be more clear?

4) This doesn't appear to handle the withdrawn applications problem in the original post. Maybe that should be moved to its own issue?

**Comment by hmanalai** [comment]
**Original comment by Ishwar Agarwal (Bitbucket: [agarwali](https://bitbucket.org/agarwali), GitHub: [agarwali](https://github.com/agarwali)).**

----------------------------------------

1. I agree that it should be part of the front page because it is not obvious the user has to go to the review page, which only appears once someone is done submitting their application.
In terms of rethinking the front page, we can use something like [this](https://dribbble.com/shots/771086-Project-Timeline-Status-Elements). Also, we once discussed that we should be checking how much of the application the user has completed. So this can be tied up with dashboard stuff.
2. The review page (currently the done page) can use an optional parameter when the user requests from the home page link.
3. If used the solution in 1) then we could take care of this.
4. Withdrawn should be a separate issue.

This also reminds me that if a user just is halfway through the application we shouldn't probably make them go to the first page everytime.

**Comment by hmanalai** [comment]
**Original comment by Guillermo ramos (Bitbucket: [guillermo_ramos](https://bitbucket.org/guillermo_ramos), GitHub: [memo330179](https://github.com/memo330179)).**

----------------------------------------

Will also move this to the front page, add explanations. I do agree that withdrawn application should be its own issue.

**Comment by hmanalai** [comment]
**Original comment by Guillermo ramos (Bitbucket: [guillermo_ramos](https://bitbucket.org/guillermo_ramos), GitHub: [memo330179](https://github.com/memo330179)).**

----------------------------------------

Issue #54 was marked as a duplicate of this issue.

**Comment by hmanalai** [comment]
**Original comment by Guillermo ramos (Bitbucket: [guillermo_ramos](https://bitbucket.org/guillermo_ramos), GitHub: [memo330179](https://github.com/memo330179)).**

----------------------------------------

Dashboard has been reworked!

**Comment by hmanalai** [comment]
**Original comment by Ishwar Agarwal (Bitbucket: [agarwali](https://bitbucket.org/agarwali), GitHub: [agarwali](https://github.com/agarwali)).**

----------------------------------------

@memo330179 the progress bar that appears on each of the form pages has yet to be implemented from what we decided last time. Is that still an issue?

**Comment by hmanalai** [comment]
**Original comment by Guillermo ramos (Bitbucket: [guillermo_ramos](https://bitbucket.org/guillermo_ramos), GitHub: [memo330179](https://github.com/memo330179)).**

----------------------------------------

I added bread crumbs for now. We can test and decide if they do the trick.

**Comment by hmanalai** [comment]
**Original comment by Guillermo ramos (Bitbucket: [guillermo_ramos](https://bitbucket.org/guillermo_ramos), GitHub: [memo330179](https://github.com/memo330179)).**

----------------------------------------

![Screenshot from 2017-02-02 09-01-02.png](https://bitbucket.org/repo/e47RB5/images/1645113378-Screenshot%20from%202017-02-02%2009-01-02.png)
